### DISCUSSION TITLE
How to change title for dialogbox  when using relationshipmanager ?

### DISCUSSION THREAD

**Discussion by jigujigu** [filamentphp/filament#5722] [Help] [answered]
Hi All,

How I can change the title of add-dialog when using relationshipmanager

![image](https://user-images.githubusercontent.com/6214964/219831504-a474c328-11db-4fd3-9ba5-6b76620b7c74.png)

Thank you

**Comment by danharrin** [comment, marked as answer]
`CreateAction::make()->modalHeading('New heading')`
